.tekton/integration-tests/pipelines/lightspeed-integration-test-pipeline.yaml

```diff
@@ -61,15 +134,34 @@ spec:
                 valueFrom:
                   fieldRef:
                     fieldPath: metadata.labels['appstudio.openshift.io/component']
+              - name: KUBECONFIG
+                value: "/credentials/$(steps.get-kubeconfig.results.kubeconfig)"
+            volumeMounts:
+              - name: credentials
+                mountPath: /credentials
             image: registry.redhat.io/openshift4/ose-cli:latest
             script: |
+              echo "---------------------------------------------"
+              cat $KUBECONFIG
+              echo "---------------------------------------------"
               dnf -y install jq python3-pip
-
+              export OPERATOR_SDK_VERSION=1.36.1
+              export ARCH=$(case $(uname -m) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n $(uname -m) ;; esac)
+              export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v${OPERATOR_SDK_VERSION}
+              curl -Lo /usr/local/bin/operator-sdk ${OPERATOR_SDK_DL_URL}/operator-sdk_linux_${ARCH}
+              chmod +x /usr/local/bin/operator-sdk
+              operator-sdk version
               echo "---------------------------------------------"
-              echo ${SNAPSHOT} | jq .
+              oc new-project $(params.namespace)
+              export IMAGE_DIGEST_MIRROR_SET=https://raw.githubusercontent.com/openshift/lightspeed-operator/main/config/manager/imagedigestmirrorset.yaml
+              curl -Lo /usr/tmp/imagedigestmirrorset "$IMAGE_DIGEST_MIRROR_SET"
+              oc apply -f /usr/tmp/imagedigestmirrorset
```

Comment:
If we are using the test bundle that is already done with image substitution, we could ignore this `imagedigestmirrorset`

Reply:
yeah, lemme remove it